fairing/builders/builder.py

```diff
@@ -21,12 +22,12 @@ def get_container_builder(builder_str=None):
 
 def get_default_container_builder():
     if is_running_in_k8s():
-        return get_builder(Builders.KNATIVE)
-    return get_builder(Builders.DOCKER)
+        return get_builder(Builders.CM)
+    return get_builder(Builders.CM)
 
 
 def get_builder(builder):
-    if builder == Builders.DOCKER:
-        return DockerBuilder()
```

Comment:
ah! I had multiple issues importing docker.ApiClient from a python2 environment. All I need is a way to avoid importing it since I have no use for DockerBuilder. Any suggestions?

Reply:
fixed the issue. Only import if needed.